arroyo-sql/src/lib.rs

```diff
@@ -338,7 +338,7 @@ impl ArroyoSchemaProvider {
                 UdfDef {
                     args,
                     ret,
-                    def: function.to_token_stream().to_string(),
+                    def: body.to_string(),
```

Comment:
Note that we're now writing the whole definition to `lib.rs`. This way `use` statements don't need to be inside the function.